angular/projects/spark-angular/src/lib/components/sprk-modal/sprk-modal.component.ts

```diff
@@ -132,6 +133,16 @@ export class SprkModalComponent {
    * per page.
    */
   @Input()
+  closeAnalyticsString: string;
```

Comment:
@ka6thick54 Thanks for the PR! Let's update the description of this new Input to match the other `analyticsString` Inputs. 
/**
   * The value supplied will be assigned to the
   * `data-analytics` attribute
   * on the close button. Intended
   * for an outside library to capture data.
   */

Reply:
I ll do the test and description.